packages/tools/storybook/.storybook/manager.js

```diff
@@ -2,6 +2,45 @@ import { addons } from '@storybook/addons';
 import theme from './ods.theme';
 import React from 'react';
 
+import { releases } from '../public/releases.js';
+
+const createHeaderElement = (element) => {
+  const timer = setInterval(function(){
```

Comment:
😧 is this really necessary?
can't you just add a dom loaded listener to start all your changes instead?